### PR TITLE
Release v1.2.6

### PR DESCRIPTION
**Pull request by bryanculver**
In preparation for `v1.2.6` release on February 22nd, merge `develop` into `main`.

### CODE REVIEW

File: nautobot/docs/release-notes/version-1.2.md [hunk 1]
```diff
@@ -140,6 +140,27 @@ All models that have `slug` fields now use `AutoSlugField` from the `django-exte
 
 Just as with the UI, the `slug` can still always be explicitly set if desired.
 
+## v1.2.6 (2022-MM-DD)
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-1.2.md:143]:
```suggestion
## v1.2.6 (2022-02-22)
```

File: nautobot/docs/release-notes/version-1.2.md [hunk 2]
```diff
@@ -140,6 +140,27 @@ All models that have `slug` fields now use `AutoSlugField` from the `django-exte
...
+
+### Added
+
+- [#1353](https://github.com/nautobot/nautobot/issues/1353) - Added UI for deleting previously uploaded images when editing a DeviceType.
+
+### Fixed
+
+- [#1249](https://github.com/nautobot/nautobot/issues/1249) - Fixed a timing issue where after creating a custom field with a default value and immediately assigning values to this custom field on individual objects, the custom field values could be automatically reverted to the default value.
+- [#1279](https://github.com/nautobot/nautobot/issues/1279) - Circuit terminations now render custom relationships on the circuit detail page.
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-1.2.md:152]: Possibly move this item up under `Added` instead? Debatable.

File: pyproject.toml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 name = "nautobot"
 # Primary package version gets set here. This is used for publishing, and once
 # installed, `nautobot.__version__` will have this version number.
-version = "1.2.5"
+version = "1.2.6-beta.1"
```
Comment (glennmatthews) [inline pyproject.toml:5]:
```suggestion
version = "1.2.6"
```

**Review by glennmatthews — APPROVED** [review]

File: nautobot/docs/release-notes/version-1.2.md (hunk 2 above)
Comment (bryanculver) [reply nautobot/docs/release-notes/version-1.2.md:152]: Yeah I would say that is an `Added`.

**Review by lampwins — APPROVED** [review]

**Pull request merged**